lib/index-api.js

```diff
@@ -33,6 +33,16 @@ module.exports = function(config, auth, storage) {
   app.use(expressJson5({ strict: false, limit: config.max_body_size || '10mb' }))
   app.use(Middleware.anti_loop(config))
 
+  // encode / in a scoped package name to be matched as a single parameter in routes
+  // TODO: routes should be modified to properly handle scopes, this is just a workaround
```

Comment:
It’s not a nginx bug, nginx behaves correctly. It’s just plain stupid idea to use encoded `/` in path that shouldn’t be represented as a path.